src/app/page.tsx

```diff
@@ -1,47 +1,53 @@
-import Image from "next/image";
+"use client";
+
+import { PairsList } from "./PairsList";
+import { createContext, useEffect, useState } from "react";
+import * as adex from "alphadex-sdk-js";
+import { PairInfo } from "./PairInfo";
+
+export const AdexStateContext = createContext(null);
 
 // more components here: https://daisyui.com/components/
 
+adex.init(); //Connect to alphadex websocket
```

Comment:
Move this line to just below all the imports. Although it is not strictly necessary, it is best to initialise the SDK first before using any of the variables.